Sources/Components/Expandable/Expandable.swift

```diff
@@ -0,0 +1,142 @@
+     Consider it for long content that may overwhelm users if displayed all at once.
+
+     **Usage**
+
+     Provide the `Warp.Expandable` with:
+
+     - A title.
+     - A subtitle.
+     - A `Warp.ExpandableStyle`, choose from:
+        - `default`: for a non bordered expandable.
+        - `box`: for a bordered expandable with rounded corners.
+        - `boxBleed`: for a bordered expandable with sharp corners.
+     - A boolean `isAnimated` to determine if the expand/collapse transition is animated or not. **Optional** _default value is `true` if none is specified_.
+     - A `ColorProvider`. **Optional:** _default is read from `Config.colorProvider` if none is specified_.
+     */
+    public struct Expandable: View {
+        /**
+        **Collapsed by default**
+
+         Expendables begins in the collapsed state with all content panels closed.
+         Starting in a collapsed state gives the user a high-level overview of the available information.
+        */
+        @State private var isExpanded: Bool = false
+
+        let title: String
+        let subtitle: String
+        let style: Warp.ExpandableStyle
+        let isAnimated: Bool
+
+        let colorProvider: ColorProvider
+
+        public init(
+            title: String,
+            subtitle: String,
+            style: Warp.ExpandableStyle,
+            isAnimated: Bool = true,
+            colorProvider: ColorProvider = Config.colorProvider
+        ) {
+            self.title = title
+            self.subtitle = subtitle
+            self.style = style
+            self.isAnimated = isAnimated
+            self.colorProvider = colorProvider
+        }
+
+        public var body: some View {
+            VStack(spacing: Warp.Spacing.spacing200) {
+                alwaysVisibleView
+
+                if isExpanded {
+                    expandableView
+                }
+            }
+            .frame(maxWidth: .infinity)
+            .padding(Warp.Spacing.spacing200)
+            .background(style.backgroundColor(using: colorProvider))
+            .cornerRadius(style.cornerRadius)
+        }
+
+        @ViewBuilder private var alwaysVisibleView: some View {
+            HStack {
+                Warp.Text(
+                    title,
+                    style: .bodyStrong,
+                    color: colorProvider.expandableTitleText
+                )
+
+                if style != .default {
+                    Spacer()
+                }
+
+                Image("icon-expandable-chevron", bundle: .module)
+                    .renderingMode(.template)
+                    .foregroundColor(colorProvider.expandableIcon)
+                    .frame(width: 16, height: 16)
+                    .rotationEffect(.degrees(isExpanded ? 180 : 0))
+
+                if style == .default {
+                    Spacer()
+                }
+            }
+            .contentShape(Rectangle())
+            .onTapGesture {
```

Comment:
`.onTapGesture` vs `Button` I think I've seen some tweets about getting some free accessibility stuff if you use a button instead of .onTapGesture 🤔 

Do you have any pros/cons ?

Reply:
I started out with having this being a button but there was a problem with the animations looking wonky if I did so. Let me try again because I've made _a lot_ of changes since then, so maybe some of them has done something good for me.

Reply:
Ah yes...there was the problem with the button highlighting on taps even with `.buttonStyle(.plain)` set.

Fixed that with a new private `NonHighlightedButtonStyle`

Reply:
You can see how it looks in [the most recent commit](https://github.com/warp-ds/warp-ios/pull/54/commits/dca6d67569fa7aa6308c821a002f3ae19a423f3e)